### PR TITLE
SPCTR-2850 - [Forms] Getting multiple entries on email.

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
[Forms] Getting multiple entries on email.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/forms.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+		const getJsELement = document.querySelector( `${id}:not(.uagb-activated-script)` );
+		if( ! getJsELement ) return null;
+
+		// Ensures that the script only runs once on the given element (required for homepage sidebar usage case).
+		getJsELement.classList.add( 'uagb-activated-script' );
+		return getJsELement;
+	}, 
+
 	init( attr, id, post_id ) {
+		const getLottieElement = UAGBForms.getElement( id );
```
Comment (AbhijitNage123) [inline assets/js/forms.js:14]: should be `getFormsElement`

File: assets/js/forms.js [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+		const getJsELement = document.querySelector( `${id}:not(.uagb-activated-script)` );
+		if( ! getJsELement ) return null;
+
+		// Ensures that the script only runs once on the given element (required for homepage sidebar usage case).
+		getJsELement.classList.add( 'uagb-activated-script' );
+		return getJsELement;
+	}, 
+
 	init( attr, id, post_id ) {
+		const getFormsElement = UAGBForms.getElement( id ); // eslint-disable-line no-undef
```
Comment (rahulvarma722) [inline assets/js/forms.js:14]: You need to use this variable for getting element getFormsElement.

Comment (rahulvarma722) [inline assets/js/forms.js:14]: You need to use this variable for getting element getFormsElement.

File: assets/js/forms.js [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+
+		// Ensures that the script only runs once on the given element (required for homepage sidebar usage case).
+		getJsELement.classList.add( 'uagb-activated-script' );
+		return getJsELement;
+	}, 
+
 	init( attr, id, post_id ) {
-		const scope = document.querySelector( id );
-		if ( ! scope ) {
+		const scope = UAGBForms.getElement( id ); // eslint-disable-line no-undef
```
Comment (RajkiranBagal) [inline assets/js/forms.js:14]: We can remove this `eslint-disable` by just defining it in `.eslintrc.js` globals.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: assets/js/forms.js (hunk 2 above)
Comment (supriyajjadhav) [reply assets/js/forms.js]: Updated Please check

File: assets/js/forms.js [hunk 4]
```diff
@@ -1,10 +1,21 @@
 // eslint-disable-next-line no-undef
```
Comment (rahulvarma722) [inline assets/js/forms.js:1]: Please remove this also eslint comment

Comment (supriyajjadhav) [reply assets/js/forms.js:1]: Updated

**Review by rahulvarma722 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**